backend/game/cards/card.py

```diff
@@ -7,3 +7,7 @@ def __init__(self, name, cost, effect, footnote=None):
         self.cost = cost
         self.effect = effect
         self.footnote = footnote
+
+    # Card is a user defined object. Provided comparison context other than object reference.
```

Comment:
question: is this used as a part of this changelist?

Reply:
Player.remove card needs this to function